Shoko.Server/API/v3/Controllers/TreeController.cs

```diff
@@ -180,67 +181,12 @@ public ActionResult<List<Series>> GetSeriesInFilteredGroup([FromRoute] int filte
 
             return (recursive ? group.GetAllSeries() : group.GetSeries())
                 .Where(series => seriesIDs.Contains(series.AnimeSeriesID))
-                .OrderBy(OrderByAirDate)
+                .OrderByAirDate()
                 .Select(series => new Series(HttpContext, series))
                 .Where(series => series.Size > 0 || includeMissing)
                 .ToList();
         }
 
-        /// <summary>
```

Comment:
There is no need for a filter specific endpoint to get the main series for a group, if the group is in the filter then just use the normal endpoint to get the main series.